GNUmakefile

```
@@ -40,6 +40,8 @@ include $(MAKES)/shell.mk
 v ?=
 count ?= 1
 
+format:
+	golangci-lint fmt ./...
```

```suggestion
fmt:
	golangci-lint fmt ./...
```

Then add `fmt` to GO-CMDS-SKIP variable defined line 27.

Then the go.mk file will skip it

https://github.com/makeplus/makes/blob/488bced7b1d50e3addc25826d933f79d94a43416/go.mk#L28

And yours would be accepted.

I'm unsure about the format to provide multiple commands to ignore. But I feel like it should be something like 

```
GO-CMDS-SKIP = test fmt
```

Maybe with quotes.